lib/sidekiq/cli.rb

```diff
@@ -251,8 +249,6 @@ def options
     def boot_system
       ENV['RACK_ENV'] = ENV['RAILS_ENV'] = environment
 
-      raise ArgumentError, "#{options[:require]} does not exist" unless File.exist?(options[:require])
```

Comment:
I removed this check because we already have it in a proper place – validation. The only possible way to get `ArgumentError` exception – call `run` without `parse`, which I find irrelevant. Also, we always available to move validation from options parsing phase to the run phase.
https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/cli.rb#L294